drivers/adxl345/include/adxl345_params.h

```diff
@@ -48,14 +48,14 @@ extern "C" {
                                      0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x0F}
 #endif
 #ifndef ADXL345_PARAM_FULL_RES
-#define ADXL345_PARAM_FULL_RES      (1)
+#define ADXL345_PARAM_FULL_RES      (1 << 3)
 #endif
 #ifndef ADXL345_PARAM_OFFSET
 #define ADXL345_PARAM_OFFSET        { 0, 0, 0 }
 #endif
-#ifndef ADXL345_PARAM_SCALE_FACTOR
```

Comment:
Thanks for your fix. Could you also readd this #ifndef please ?
We should be good to close this afterward.